content/version/1/4/code-of-conduct.it.md

```diff
@@ -0,0 +1,83 @@
++++
+version = "1.4"
+aliases = ["/version/1/4/it"]
++++
+
+# Codice di Comportamento dei Collaboratori
+
+## Il nostro impegno
+
+Nell'interesse di promuovere un ambiente aperto ed accogliente, noi come
+collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro 
+progetto e alla nostra comunità un'esperienza libera da molestie per tutti, 
```

Comment:
e della nostra comunità

Reply:
Ok. I do not agree with you on this point: I think it is a "complemento di termine" so it is better   "collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro progetto e alla nostra comunità".